apps/internal/base/base.go

```diff
@@ -219,6 +219,9 @@ func (b Client) AuthCodeURL(ctx context.Context, clientID, redirectURI string, s
 	if authParams.CodeChallengeMethod != "" {
 		v.Add("code_challenge_method", authParams.CodeChallengeMethod)
 	}
+	if authParams.LoginHint != "" {
```

Comment:
A simple adition to this feature would be to expose the login_hint to the `GetAuthorizationUrl` API on confidential client

Reply:
Thanks for pointing this out! I've added the option to the public and confidential URL builders.